### PR TITLE
fix(jsx): prevent bootstrap from injecting props to react primitive

### PR DESCRIPTION
**Pull request by BerkeleyTrue**
This removes a react warning

### CODE REVIEW

File: common/app/utils/No-Props-Passthrough.jsx [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+// use when passing a react primitive element as a child to a
+// react-boostrap component that will inject props
+// using cloneElement
+export default function NoPropsPassthrough({ children }) {
+  return children;
+}
+
+NoPropsPassthrough.propTypes = {
+  chilren: PropTypes.element
```
Comment (Bouncey) [inline common/app/utils/No-Props-Passthrough.jsx]: children

**Review by Bouncey — CHANGES_REQUESTED** [review]

File: common/app/utils/No-Props-Passthrough.jsx (hunk 1 above)
Comment (BerkeleyTrue) [inline common/app/utils/No-Props-Passthrough.jsx]: Nice catch. I rely on my linter way to much. Of course my dyslexia doesn't help, either.

Comment (BerkeleyTrue) [inline common/app/utils/No-Props-Passthrough.jsx]: fixed

Comment (Bouncey) [inline common/app/utils/No-Props-Passthrough.jsx]: You teach me how to code by picking up on everything, I will gloat over a typo! :smile:

**Review by Bouncey — APPROVED** [review]

**Comment by Bouncey** [comment]
## LGTM :+1:

**Pull request merged**